### ISSUE TITLE
Case layers order

### ISSUE THREAD

**Issue by d1nuc0m** [floookay/adelheid#4]
I was trying to understand the assembly order for the lasercut layers, but I have doubts

![lopro_layers_colored](https://github.com/floookay/adelheid/assets/72998205/92885054-13c9-41dd-aeef-ef33fb949c00)

In relation to the scheme above, layers from lower to upper:

- Low profile: A-B-F-C-E-D
    - Is this correct? (I'm quite confident of this one, but better safe than sorry)
- High profile: A-B-F-C-E-I-K-J-L-D
    - Is this correct?
    - If so, where should G and H go?

**Comment by floookay** [comment]
for the low profile it's: `B-A-F-C-E-D`  
for the high profile it's: `B-A-F-C-E-D-J-L`  
`G`, `H` as well as `I` and `K` are duplicates. I cut some text in the second `G` for example, but it didn't turn out that visible. The duplicate `I` and `K` pieces are useful, because these easily snap in the corners during transport or if you're not careful handling them.

**Comment by d1nuc0m** [comment]
Thank you, if I understood the order, new piece in #3 should be ok. Anyway, considered adding layered case assembly instructions to the repository? I would have made a mess without asking 😅

**Comment by floookay** [comment]
Yeah, that's a good idea. I'll have a look at it on the weekend. Or if you want you can create a PR with your sketch as well, I'll merge it.
